ui/common/src/main/res/values/strings.xml

```diff
@@ -868,6 +868,8 @@
   <string name="promotions_perks_button">Perks</string>
   <string name="promotions_vouhers_button">Vouchers</string>
   <string name="promotions_tooltip">Earn free AppCoins Credits with our promotions!</string>
+  <string name="promotions_bank_transfer_title">Easy Bank Transfer</string>
```

Comment:
@carlosgarridoaptoide the capitalisation of this payment method is 'Easy bank transfer' at the moment, see here: https://api.catappult.io/broker/8.20240801/methods?channel=ANDROID&transaction.type=TOPUP&country=DE

shouldn't it be the same on both sides?

Reply:
Amended ;)